runtime/jcl/common/thread.cpp

```diff
@@ -518,6 +518,34 @@ Java_java_lang_Thread_registerNatives(JNIEnv *env, jclass clazz)
 #if defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT)
 	clearNonZAAPEligibleBit(env, clazz, natives, numNatives);
 #endif /* J9VM_OPT_JAVA_OFFLOAD_SUPPORT */
+}
+
+/* private native void notifyJvmtiMountBegin(boolean firstMount); */
+void JNICALL
+Java_java_lang_VirtualThread_notifyJvmtiMountBegin(JNIEnv *env, jclass clazz, jobject thread, jboolean firstMount)
```

Comment:
for the new functions,
- add function prototype in `jclprots.h`; and
- add function description in doxygen format.

Reply:
They seem to be auto-generated in the build directory: `java_lang_Thread.h`. There is also an auto-generated version for `java_lang_VirtualThread.h`. if we are using the generated headers, we don't need to add prototypes in `jclprots.h`.

@fengxue-IS do you know how these headers are generated?

Reply:
> how these headers are generated?

[JEP 313](https://openjdk.org/jeps/313) deprecated/removed `javah`, now the c-headers files are generated at `HEADERS` - https://github.com/ibmruntimes/openj9-openjdk-jdk19/blob/ed7ee455ccc9810a978d8e1c81babd4d471e33cf/make/common/JavaCompilation.gmk#L476-L488